inst/examples/app/instrumentation/app.R

```diff
@@ -5,12 +5,29 @@ library(shiny.telemetry)
 library(dplyr)
 library(config)
 
+secondary_tab_ui <- function(id, label = "Counter") {
```

Comment:
Just for consistency and since this is UI for the counter, maybe we can call this `counter_ui` instead.